### ISSUE TITLE
Env overriders creates objects instead of arrays on numeric index

### ISSUE THREAD

**Issue by unlucio** [namshi/reconfig#16]
when overriding confs from environment, when we get something like:
```bash
MYPREFIX_b_0=value
```
and b is not previously defined, reconfig ends up creating
```javascript
config.b = {0: value}
```
instead of
```javascript
config.b =[value];
```

**Comment by unlucio** [comment]
mumble, I noticed too but I think this is a vpo's bug :P

**Comment by unlucio** [comment]
it was VPO's fault :dancer: fixed here:

https://github.com/unlucio/vpo/pull/7

**Comment by unlucio** [comment]
the bug gets fixed using vpo 1.1.1 :)
